src/styles/_header.scss

```diff
@@ -0,0 +1,54 @@
+.header {
+  padding: 10px 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  &_link_logo {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  &_item:not(:last-child) {
+    margin-right: 20px;
+  }
+}
+
+.nav_link {
+  font-size: 12px;
+  text-transform: uppercase;
+  color: $mainColor;
```

Comment:
The variable `$mainColor` is used here, but it is not defined within this file. Ensure that it is imported or defined elsewhere in your SCSS files.